src/tests/comment.test.js

```diff
@@ -204,4 +205,79 @@ describe('comment tests', () => {
         done();
       });
   });
+
+  it('should display an error message in case a accomodation does not have a comment', (done) => {
+    chai
+      .request(app)
+      .get(`/api/v1/accommodations/${accomodationDatabaseData[3].id}/comments?page=1`)
+      .set('token', `Bearer ${anauthorizedToken}`)
+      .end((err, res) => {
+        res.should.have.status(404);
+        res.body.should.have.be.a('object');
+        res.body.should.have.property('error').eql('No comment yet');
+        done();
+      });
+  });
+
+  it('should display all accommodation comments', (done) => {
+    chai
+      .request(app)
+      .get(`/api/v1/accommodations/${accomodationDatabaseData[0].id}/comments?page=1&limit=1`)
+      .set('token', `Bearer ${token}`)
+      .end((err, res) => {
+        res.should.have.status(200);
+        res.body.should.have.be.a('object');
+        res.body.should.have.property('message').eql('success');
+        res.body.data.commenterInfo.should.have.property('id').eql(UserDatabaseData[2].id);
+        res.body.data.commenterInfo.should.have.property('firstName').eql(UserDatabaseData[2].firstName);
+        res.body.data.commenterInfo.should.have.property('lastName').eql(UserDatabaseData[2].lastName);
+        res.body.data.commenterInfo.should.have.property('profileImage');
+        res.body.data.comment.rows.should.have.be.a('array');
+        res.body.data.comment.rows[0].should.have.property('id').eql(comment[4].id);
+        res.body.data.comment.rows[0].should.have.property('subjectType').eql(comment[4].subjectType);
+        res.body.data.comment.rows[0].should.have.property('commentorId').eql(comment[4].commentorId);
+        res.body.data.comment.rows[0].should.have.property('comment').eql(comment[4].comment);
+        done();
+      });
+  });
+
+  it('should check if a accomodation exist', (done) => {
+    chai
+      .request(app)
+      .get('/api/v1/accommodations/0/comments?page=1')
+      .set('token', `Bearer ${token}`)
+      .end((err, res) => {
+        res.should.have.status(404);
+        res.body.should.have.be.a('object');
+        res.body.should.have.property('error').eql('Accommodation not found');
+        done();
+      });
+  });
+
+  it('should display error message in case a comment does not exist', (done) => {
+    chai
+      .request(app)
+      .delete(`/api/v1/accommodations/${accomodationDatabaseData[0].id}/comments/0`)
+      .set('token', `Bearer ${token}`)
+      .end((err, res) => {
+        res.should.have.status(404);
+        res.body.should.have.be.a('object');
+        res.body.should.have.property('error').eql('Comment not found');
+        done();
+      });
+  });
+
+  it('comment should check if a user is allowed to provided a feedback on the accommodation', (done) => {
```

Comment:
@shemaeric1234 here you're describing what a test case is doing. So it not a comment. Always starts with "should". Kindly update elsewhere applicable.